File: java/delivery-api/GetElement.java
// DocSection: delivery_api_get_element
// Tip: Find more about Java SDK at https://docs.kontent.ai/java
import kentico.kontent.delivery.*;

// Initializes a DeliveryClient
DeliveryClient client = new DeliveryClient("<YOUR_PROJECT_ID>");

// Gets the "title" content element from the "article" type using a simple request
CompletionStage<Element> element = client.getContentTypeElement("article", "title");
// To use the code for Android projects, see http://docs.kontent.ai/android
// EndDocSection

File: ruby/delivery-api/GetElement.rb
# DocSection: delivery_api_get_element
# Tip: Find more about Ruby SDKs at https://docs.kontent.ai/ruby
require 'delivery-sdk-ruby'

delivery_client = Kentico::Kontent::Delivery::DeliveryClient.new project_id: '<YOUR_PROJECT_ID>'
delivery_client.element('coffee', 'processing').execute do |response|
  ele = response.element
end
# EndDocSection